source/Sailfish/Execution/SailFishTestExecutor.cs

```diff
@@ -43,10 +43,15 @@ private static bool FilterEnabledType(IEnumerable<Type> testTypes, out Type[] en
             return rawResults;
         }
 
+        var testIndex = 0;
+        var totalTestCount = enabledTestTypes.Length;
+
+        Console.WriteLine($"Discovered {totalTestCount} enabled test type{(totalTestCount == 1 ? "" : "s")}");
```

Comment:
What do you think about using `logger.Information` instead of `Console.WriteLine`?